src/app/locale/translations/en.json

```diff
@@ -197,7 +197,11 @@
             "checkoutcom_document_invalid_error_ideal": "BIC must be 8 or 11 characters",
             "checkoutcom_document_invalid_error_oxxo": "CURP must be at most 18 characters",
             "checkoutcom_document_invalid_error_qpay": "National ID must be at most 32 characters",
-            "checkoutcom_sepa_mandate_disclaimer":"By accepting this mandate form, you authorize {storeName} and Checkout.com, our payment service provider, to send instructions to your bank to debit your account, and your bank to debit your account in accordance with those instructions. You are entitled to a refund from your bank under the terms and conditions of your agreement with your bank. A refund must be claimed within 8 weeks starting from the date on which your account was debited.",
+            "checkoutcom_fawry_customer_mobile_invalid_error": "A valid Mobile Number is required",
+            "checkoutcom_fawry_customer_mobile_label": "Mobile Number",
+            "checkoutcom_fawry_customer_email_invalid_error": "Email address must be valid",
```

Comment:
Probably make the error translations consistent for both mobile and email?
either `a valid X is required` or `X must be valid`

Looking at other translations, the latter format seems to be more common, e.g `customer.email_invalid_error: Email address must be valid`